wikimedia/wikipedia-ios
# Replace @Wikipedia Twitter handle with Wikipedia name

**Phabricator:** 
https://phabricator.wikimedia.org/T225180
### Notes
* I replaced the @Wikipedia Handle with just Wikipedia 
* Unsure if translation needs to be done

### Test Steps
1. Open an article on Wikipedia 
2. Share article to twitter
3. Check for the absence of @Wikipedia handle
